addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -923,7 +923,7 @@ export function makeActionManager(env, router = _router) {
             if (size) {
                 actionDialogProps.size = size;
             }
-
+            actionDialogProps.footer = action.context.footer ?? actionDialogProps.footer;
```

Comment:
What are the fixes in web? Could they be separate PR/fix?

Reply:
remove the footer from it. it was not working well previously and paju wants to get rid of the default confirm footer.